### PR TITLE
Create lu-rcs.json

### PR DESCRIPTION
**Pull request by titiolukole**

### CODE REVIEW

File: lists/lu/lu-rcs.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "data_not_available"
+    ],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "no_license",
+    "licenseDetails": ""
+  },
+  "meta": {
+    "source": "Desk Research",
+    "lastUpdated": "2022-09-05"
```
[the remark this answers is not in the document]
Comment (kd-ods) [reply lists/lu/lu-rcs.json:41]: Rejecting suggestion.

File: lists/lu/lu-rcs.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "coverage": [
+    "LU"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "government_agency"
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:15]:
```suggestion
    "company"
```

Comment (kd-ods) [reply lists/lu/lu-rcs.json:15]: This field should let people know what kinds of organisations have this type of identifier. So: those which are constituted as companies (as I understand it). http://docs.org-id.guide/en/latest/schema/

File: lists/lu/lu-rcs.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  ],
+  "sector": [],
+  "code": "LU-RCS",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:21]: If RCS numbers are issued at the point that an entity is registered as a company then this value should be "primary"

File: lists/lu/lu-rcs.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "",
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:25]:
```suggestion
    "publicDatabase": "https://www.lbr.lu/mjrcs/jsp/DisplayConsultDocumentsActionNotSecured.action",
```

Comment (kd-ods) [reply lists/lu/lu-rcs.json:25]: This should be here users go to search or access this list.

File: lists/lu/lu-rcs.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  "structure": [
+    "government_agency"
+  ],
+  "sector": [],
+  "code": "LU-RCS",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
+  "access": {
+    "availableOnline": false,
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:23]:
```suggestion
    "availableOnline": true,
```

Comment (kd-ods) [reply lists/lu/lu-rcs.json:23]: RCS numbers can be searched for on the website.

File: lists/lu/lu-rcs.json [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    "onlineAccessDetails": null,
+    "publicDatabase": "",
+    "guidanceOnLocatingIds": "",
+    "exampleIdentifiers": "",
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:27]: Please add some examples. Like 'B189486'

File: lists/lu/lu-rcs.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    "languages": []
```
Comment (kd-ods) [inline lists/lu/lu-rcs.json:28]: Looks like the website is available in French, German and English.

**Review by kd-ods — CHANGES_REQUESTED** [review]
Hi @titiolukole - I've requested and suggested some changes. Ping me directly if you need any support around editing this entry.

**Comment by kd-ods** [comment]
Hi @titiolukole -
It looks like you've accepted my suggestions where I made them. But for the unresolved requests, you'll need to edit the json file again. So open the 'Files changed' tab (preferably in a separate tab so you can still refer to my comments) and click on the three dots at the top right of the files content to choose the 'Edit file' option. You'll then be able to edit the json by hand.  See the 'Addressing review comments on a GitHub pull request' section of the guidance doc (which I've just added a bit to).

Email me if you need to go through this on a call.

File: lists/lu/lu-rcs.json (hunk 6 above)
Comment (titiolukole) [reply lists/lu/lu-rcs.json]: We have given 2 example identifiers

**Review by kd-ods — APPROVED** [review]

**Pull request merged**